test/Common.h

```diff
@@ -31,21 +33,27 @@ namespace solidity::test
 #ifdef _WIN32
 static constexpr auto evmoneFilename = "evmone.dll";
 static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.4.1/evmone-0.4.1-windows-amd64.zip";
+static constexpr auto heraFilename = "hera.dll";
+static constexpr auto heraDownloadLink = "https://github.com/ewasm/hera/archive/v0.3.0.tar.gz";
 #elif defined(__APPLE__)
 static constexpr auto evmoneFilename = "libevmone.dylib";
 static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.4.1/evmone-0.4.1-darwin-x86_64.tar.gz";
+static constexpr auto heraFilename = "libhera.dylib";
+static constexpr auto heraDownloadLink = "https://github.com/ewasm/hera/releases/download/v0.3.0/hera-0.3.0-darwin-x86_64.tar.gz";
 #else
 static constexpr auto evmoneFilename = "libevmone.so";
 static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.4.1/evmone-0.4.1-linux-x86_64.tar.gz";
+static constexpr auto heraFilename = "libhera.so";
+static constexpr auto heraDownloadLink = "https://github.com/ewasm/hera/releases/download/v0.3.0/hera-0.3.0-linux-x86_64.tar.gz";
 #endif
 
-
 struct ConfigException : public util::Exception {};
 
 struct CommonOptions: boost::noncopyable
 {
-	boost::filesystem::path evmonePath;
+	std::vector<boost::filesystem::path> vmPaths;
 	boost::filesystem::path testPath;
+	bool enableEwasm = false;
```

Comment:
Maybe the "enable" part is redundant?

Reply:
Maybe `enableEwasm` is not the best name. I introduced `--ewasm` to the command-line options, if present `enableEwasm` is set to `true`. If enabled, a search for the hera evmc library will be done by using `envOrDefaultPath`. With this the ewasm tests can just be enable with `--ewasm` in our CI, because the libraries are there already in place. It is true that this is somehow redundant with specifying `--vm <path-to-hera>`, but I think it simplifies the testscripts.

Reply:
I actually meant that this can be renamed to `bool ewasm`, but it's fine.

Reply:
Ah, yes.. now I see what you meant :)